amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/raml/RamlDocumentParser.scala

```diff
@@ -215,20 +219,23 @@ abstract class RamlDocumentParser(root: Root)(implicit val ctx: RamlWebApiContex
       entries => {
         val endpoints = mutable.ListBuffer[EndPoint]()
         entries.foreach(entry => ctx.factory.endPointParser(entry, api.withEndPoint, None, endpoints, false).parse())
-        api.set(WebApiModel.EndPoints, AmfArray(endpoints))
+        api.set(WebApiModel.EndPoints, AmfArray(endpoints), Annotations(SynthesizedField()))
         ctx.mergeAllOperationContexts()
       }
     )
     RamlServersParser(map, api).parse()
     val idCounter         = new IdCounter()
-    val RequirementParser = RamlSecurityRequirementParser.parse(api.withSecurity, idCounter) _
+    val RequirementParser = RamlSecurityRequirementParser.parse(api.id, idCounter) _
     map.key("securedBy", (WebApiModel.Security in api using RequirementParser).allowingSingleValue)
     map.key(
       "documentation",
       entry => {
-        api.set(WebApiModel.Documentations,
-                AmfArray(UserDocumentationsParser(entry.value.as[Seq[YNode]], ctx.declarations, api.id).parse()),
-                Annotations(entry))
+        api.set(
+          WebApiModel.Documentations,
+          AmfArray(UserDocumentationsParser(entry.value.as[Seq[YNode]], ctx.declarations, api.id).parse(),
```

Comment:
OOF - `entry.value.as[Seq[YNode]]` shouldn't be as[YSequence]?

Reply:
YSequence is an YPart that contains a Seq[YNode]. If you going to use only the values, you can do as[Seq[YNode]].
In fact, that code did not change.
This is the discussion of YNode vs YNode.value in annotations.